neet/boolean/logicnetwork.py

```diff
@@ -210,20 +220,128 @@ def update(self, net_state, index=None, pin=None):
             [0, 0, 0]
         """
         if net_state not in self.state_space:
-            raise ValueError("the provided state is not in the network's state space")
+            raise ValueError(
+                "the provided state is not in the network's state space")
```

Comment:
My auto-format plugin reformats any line longer than 79 characters into multiple lines according to PEP8.